ios/facebook_app_events.podspec

```diff
@@ -12,7 +12,7 @@ Flutter plugin for Facebook Analytics and App Events
   s.source_files = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Flutter'
-  s.dependency 'FBSDKCoreKit', '~> 5.0'
+  s.dependency 'FBSDKCoreKit', '~> 5.5.0'
```

Comment:
According to the documentation, this would actually impose a more strict limit to `>= 5.5.0` and `< 5.6.0` 

https://guides.cocoapods.org/using/the-podfile.html

![image](https://user-images.githubusercontent.com/2051607/85980232-95967800-ba14-11ea-992b-dd92ee78e453.png)